Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "PhyloPlots"
 uuid = "c0d5b6db-e3fc-52bc-a87d-1d050989ed3b"
 license = "MIT"
-version = "1.0.1"
+version = "1.1.0"
```

Comment:
I'm fine with 1.1.0, but out of curiosity, what made you change your mind and not go to 2.0.0 ?
(As it is a non backward compatible change, see https://github.com/JuliaPhylo/PhyloPlots.jl/pull/20#discussion_r945982672)

Reply:
Ah, indeed, it should be 2.0.0. When starting this PR, I merely wanted a version compatible with the new PhyloNetworks v0.17.0. Later I noticed the open issue #21 and thought that it was a good time to act on it, introducing a breaking change.

I also agree with removing "phylogram", but it would be nice to keep the term "rightwards" to be explicit that these plots won't convey the "semidirected" nature of networks like "circular" trees can do.

Reply:
Thank you, looks good to me !